triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/ServiceTest.scala

```diff
@@ -468,21 +468,22 @@ class ServiceTest extends AsyncFlatSpec with Eventually with Matchers with Postg
       } yield succeed
   }
 
-  it should "not act on a stop request without a token" in withHttpService(None) {
-    (uri: Uri, client: LedgerClient, ledgerProxy: Proxy) =>
-      val uuid: String = "ffffffff-ffff-ffff-ffff-ffffffffffff"
-      val req = HttpRequest(
-        method = HttpMethods.DELETE,
-        uri = uri.withPath(Uri.Path(s"/v1/stop/$uuid")),
-      )
-      for {
-        resp <- Http().singleRequest(req)
-        body <- responseBodyToString(resp)
-        JsObject(fields) = body.parseJson
-        _ <- fields.get("status") should equal(Some(JsNumber(422)))
-        _ <- fields.get("errors") should equal(
-          Some(JsArray(JsString("missing Authorization header with OAuth 2.0 Bearer Token"))))
-      } yield succeed
+  it should "give a 401 response for a stop request without an authorization header" in withHttpService(
```

Comment:
```suggestion
  it should "give an Unauthorized error for a stop request without an authorization header" in withHttpService(
```
since we are avoiding hardcoding status codes in the tests.